dvc/command/metrics.py

```diff
@@ -46,6 +47,24 @@ def run(self):
             )
 
             show_metrics(metrics, self.args.all_branches, self.args.all_tags)
+
+            found = set(
+                itertools.chain.from_iterable(
+                    metric.keys() for metric in metrics.values()
+                )
+            )
+
+            missing = set(self.args.targets) - found
```

Comment:
and what if target was a directory with -R? You won't find it in keys.

Reply:
:thinking: that's true!